android/app/src/main/AndroidManifest.xml

```diff
@@ -8,22 +8,12 @@
     <uses-permission
         android:name="android.permission.READ_PHONE_STATE"
         tools:node="remove" />
-    <uses-permission android:name="android.permission.CAMERA" />
```

Comment:
Let's go farther and explicitly say we don't want a `<uses-permission>` element like this one. The `READ_PHONE_STATE` element above this provides an example.

That way, if some future dependency wants to introduce such an element for some reason, it won't actually show up in the manifest. That's useful because we have this new constraint that having this permission in the manifest would be bad, potentially causing an issue that's annoying for users and subtle to debug.

(And OTOH if that library does actually need the permission, hopefully we'll have a better shot at debugging that -- because it'll be a breakage in the thing we're actively working on and thinking about -- than we would at it subtly causing the breakage we're thinking about now.)